api/v1beta1/eventtrigger_webhook.go

```diff
@@ -47,13 +57,25 @@ var _ webhook.Validator = &EventTrigger{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (e *EventTrigger) ValidateCreate() error {
 	eventtriggerlog.Info("validate create", "name", e.Name)
-	return nil
+
+	allErrs := e.validateVerticaDBRefSpec()
+	if allErrs == nil {
+		return nil
+	}
+
+	return apierrors.NewInvalid(schema.GroupKind{Group: Group, Kind: EventTriggerKind}, e.Name, allErrs)
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
 func (e *EventTrigger) ValidateUpdate(old runtime.Object) error {
 	eventtriggerlog.Info("validate update", "name", e.Name)
-	return nil
+
+	allErrs := e.validateVerticaDBRefSpec()
+	if allErrs == nil {
+		return nil
+	}
+
+	return apierrors.NewInvalid(schema.GroupKind{Group: Group, Kind: EventTriggerKind}, e.Name, allErrs)
```

Comment:
We can replace the first function with `GkET`. Similar comment in other spots in this file.